src/components/Label/TopicPageLabel.js

```diff
@@ -21,20 +21,21 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     marginStart: 20,
     alignSelf: 'flex-start',
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
     height: normalize(29)
   },
   domainText: {
     fontSize: normalizeFontSize(18),
     fontFamily: fonts.inter[600],
     lineHeight: normalize(29),
     fontWeight: 'bold',
-    textAlign: 'left'
+    textAlign: 'left',
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `color: COLORS.white2` is a good move for improving the visibility of the text against the darker background. However, make sure this color change doesn't affect the readability on different screens or under different lighting conditions.

```diff
+    color: COLORS.white2
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->